crates/ostree-ext/src/tar/export.rs

```diff
@@ -75,13 +75,13 @@ fn map_path_inner<'p>(
 }
 
 /// Convert /usr/etc back to /etc
-fn map_path(p: &Utf8Path) -> std::borrow::Cow<Utf8Path> {
+fn map_path<'a>(p: &'a Utf8Path) -> std::borrow::Cow<'a, Utf8Path> {
```

Comment:
The majority of these were recommendations by cargo itself, and passed tests, so I think they are okay.